worlds/musedash/Options.py

```diff
@@ -154,24 +155,26 @@ class TrapCountPercentage(Range):
     display_name = "Trap Percentage"
 
 
-class IncludeSongs(ItemSet):
+class SongSet(OptionSet):
+    valid_keys = sorted([x for x in SONG_DATA.keys()])
```

Comment:
You might want to unsort these, that way webhost will auto-sort them instead. Currently it lists all lowercase names after the uppercase ones, but letting webhost sort, it alphabetizes them grouped together by capitalization
```suggestion
    valid_keys = SONG_DATA.keys
```

Reply:
Oops forgot it did that. Sorted it out.